Zend/zend_alloc.c

```diff
@@ -173,6 +171,68 @@ static size_t _real_page_size = ZEND_MM_PAGE_SIZE;
 	} while (0)
 #endif
 
+#ifdef __SANITIZE_ADDRESS__
```

Comment:
Nit: Nesting in preprocessor instructions happens with `#<nr of spaces equal to nesting level>stuff`. So the `#if 0` below would need to be `# if 0`. The defines inside the `#if 0` would need to be `#  define` etc.